app/services/unverified-reports.js

```diff
@@ -0,0 +1,60 @@
+import { computed } from "@ember/object";
+import Service, { inject as service } from "@ember/service";
+import moment from "moment";
+
+const INTERVAL_DELAY = 10 * 60000; // 10 Minutes
+
+/**
+ * Unverified reports service
+ *
+ * This service polls reports of last month that need to be verified
+ * by the current user.
+ *
+ * @class UnverifiedReportsService
+ * @extends Ember.Service
+ * @public
+ */
+export default Service.extend({
+  store: service(),
+  session: service(),
+  notify: service(),
+
```

Comment:
I recommend adding the property "amountReports" with an initial value of zero.

This way you don't have weird errors/behavior if the property is read before the first polling is over.